crates/aptos/src/common/types.rs

```diff
@@ -48,6 +48,7 @@ use std::{
 };
 use thiserror::Error;
 
+pub const USER_AGENT: &str = concat!("aptos-cli / ", env!("CARGO_PKG_VERSION"));
```

Comment:
I see in your previous examples there is no space on each side of the slash. Can we do that here too?
```suggestion
pub const USER_AGENT: &str = concat!("aptos-cli/", env!("CARGO_PKG_VERSION"));
```

Reply:
Yep, David removed the slash on both sides now which is the de facto way that it is done.